server/jobs/index.js

```diff
@@ -50,12 +50,13 @@ const jobs = {
 export default function executeJobs(type) {
   const { jobList, placementStatus, automationResult } = jobs[type];
   let fetchPlacementError;
-  return fetchNewPlacements(placementStatus, 1)
+  return fetchNewPlacements(placementStatus)
     .catch(() => {
       fetchPlacementError = 'error';
       setTimeout(() => executeJobs(type), ms('5m'));
     })
     .then(async (newPlacements) => {
+      console.log('new placements', newPlacements);
```

Comment:
You can remove this `console.log` when done with debugging

Reply:
Sure, I will get rid of it.